pkg/manager/member/tidb_member_manager.go

```diff
@@ -390,12 +390,18 @@ func getTiDBConfigMap(tc *v1alpha1.TidbCluster) (*corev1.ConfigMap, error) {
 	}
 
 	plugins := tc.Spec.TiDB.Plugins
-	startScript, err := RenderTiDBStartScript(&TidbStartScriptModel{
-		ClusterName:     tc.Name,
+	tidbStartScriptModel := &TidbStartScriptModel{
 		EnablePlugin:    len(plugins) > 0,
 		PluginDirectory: "/plugins",
 		PluginList:      strings.Join(plugins, ","),
-	})
+	}
+
+	if len(tc.Spec.PdAddress) > 0 {
+		tidbStartScriptModel.Path = strings.Join(tc.Spec.PdAddress, ",")
```

Comment:
The `path` cannot include schemes like http or https, we have to remove the scheme info for TiDB.

Reply:
OK